pkg/utils/handle_errors.go

```diff
@@ -28,10 +28,7 @@ func HandleErrorWithStatus(logger framework.Logger, c *gin.Context, statusCode i
 }
 
 // list static errors to filter
-var exceptStaticError = []error{
-	gorm.ErrRecordNotFound,
-	api_errors.ErrInvalidUUID,
-}
```

Comment:
@rodanr 
It's better to revert this change. 
We don't want to send these to sentry as these errors are not unexpected IMO. 
What do you think?

Reply:
`gorm.ErrRecordNotFound` is handled manually to have a better status code of 404
	if errors.Is(err, gorm.ErrRecordNotFound) {
		c.JSON(http.StatusNotFound, gin.H{
			"error": gorm.ErrRecordNotFound.Error(),
		})
		return
	}
	
	api_errors.ErrInvalidUUID will be automatically be prevented from sentry logging as it is of new APIError type, for whose we are preventing sentry logging for its all instances as this are predicted error from user behaviour and doesnt need to be taken care of